docs/config.rst

```diff
@@ -447,6 +447,25 @@ Features
 
    .. versionadded:: 2.2
 
+.. attribute:: LP_ENABLE_ENV_VARS
+   :type: bool
+   :value: 1
+
+   Display a user-defined set of environment variables.
+   May show if the variables are unset, set, or their actual content.
+
+   Watched variables should be added to the :attr:`LP_ENV_VARS` array.
+
+   The resulting prompt section is configured by:
+
+   - :attr:`LP_MARK_ENV_VARS_OPEN`
+   - :attr:`LP_MARK_ENV_VARS_SEP`
+   - :attr:`LP_MARK_ENV_VARS_CLOSE`
+   - :attr:`LP_COLOR_ENV_VARS_SET`
+   - :attr:`LP_COLOR_ENV_VARS_UNSET`
+
+  .. versionadded:: 2.2
```

Comment:
I bet this line is causing it. It has only 2 spaces of indentation.

Reply:
:facepalm:

Reply:
I think the linter missed it completely.